test/magpiego/server.go

```diff
@@ -19,17 +19,18 @@ const (
 )
 
 var Providers = map[string]bindings.BindingProvider{
-	"DAPRPUBSUB":      bindings.DaprPubSubBinding,
-	"KEYVAULT":        bindings.KeyVaultBinding,
-	"MONGODB":         bindings.MongoBinding,
-	"SERVICEBUS":      bindings.ServiceBusBinding,
-	"SQL":             bindings.MicrosoftSqlBinding,
-	"REDIS":           bindings.RedisBinding,
-	"DAPRSTATESTORE":  bindings.DaprStateStoreBinding,
-	"RABBITMQ":        bindings.RabbitMQBinding,
-	"DAPRSECRETSTORE": bindings.DaprSecretStoreBinding,
-	"DAPRHTTP":        bindings.DaprHttpBinding,
-	"STORAGE":         bindings.StorageBinding,
+	"DAPRPUBSUB":             bindings.DaprPubSubBinding,
+	"KEYVAULT":               bindings.KeyVaultBinding,
+	"MONGODB":                bindings.MongoBinding,
+	"SERVICEBUS":             bindings.ServiceBusBinding,
+	"SQL":                    bindings.MicrosoftSqlBinding,
+	"REDIS":                  bindings.RedisBinding,
+	"DAPRSTATESTORE":         bindings.DaprStateStoreBinding,
+	"RABBITMQ":               bindings.RabbitMQBinding,
+	"DAPRSECRETSTORE":        bindings.DaprSecretStoreBinding,
+	"DAPRCONFIGURATIONSTORE": bindings.DaprConfigurationStoreBinding,
```

Comment:
Is there a reason why we added the ConfigStore to this specific location in the map? Could we also have added that to the end of the map?

Reply:
No specific reason—was the intended order based on the components' creation dates? I'll put it at the end